site.yml

```diff
@@ -159,6 +159,30 @@
       tags: ['openstack', 'nova', 'data']
       when: not ironic.enabled|bool
 
+- name: ceph monitors
+  gather_facts: force
```

Comment:
Do any of the tasks in ceph-common or ceph-monitor require variable data that's generated via fact gathering from other hosts, outside the hosts in the group listed for this play? That's the only reason to force the gathering of facts, it forces it across the entire inventory.